package.json

```diff
@@ -38,6 +38,9 @@
     "@types/vinyl": "^1.1.29",
     "@types/vinyl-fs": "0.0.28",
     "@types/yeoman-generator": "0.0.30",
+    "bower": "^1.8.0",
```

Comment:
Since we're mucking about with private fields of a bower object, we should pin to a specific version